src/posts/paper/14-research-sources.ts

```diff
@@ -40,19 +40,70 @@ This is a highly speculative number. [Li et al.](https://www.ncbi.nlm.nih.gov/pm
+
+For well-fitted and well-sealed N95 masks (no outflow valves, tight-fitting elastic headbands, and no beards) we draw on the following findings. Many studies of the physical properties of the masks (as opposed to real-world usage data) show 85% or more protection: [Mueller et al.](https://www.cell.com/matter/pdf/S2590-2385(20)30364-7.pdf) showed 90% of small particles were blocked (less than 0.3 micron); [Zhao et al.](https://pubs.acs.org/doi/10.1021/acs.nanolett.0c02211) showed 95% blocked; [Konda et al.](https://pubs.acs.org/doi/10.1021/acsnano.0c03252) showed 85% protection with no gap; [van der Sande et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2440799/) show multiple measurements above 85x (!) protection; [Lai et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/) show that the protection from a fully-sealed mask is >89%, and drops to >80% with small leakage. Meta-analyses of real-world wearing data provide harder-to-interpret data: we see a risk ratio of 0.12 in Fig 5 of [Offeddu et al.](https://pubmed.ncbi.nlm.nih.gov/29140516/); other meta-analyses of specifically influenza ([Long et al.](https://pubmed.ncbi.nlm.nih.gov/32167245/) and [Radonovich et al.](https://jamanetwork.com/journals/jama/fullarticle/2749214)) suggest much lower protection factors, but in Radonovich et al. only 90% of the subjects "sometimes" or "always" wore the mask (and how good is a mask you only "sometimes" wear at work as a health-care worker?), and are not well-designed to distinguish between health-care workers contracting influenza at work versus during their daily life, so we omit these. [Chu et al.](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)31142-9/fulltext) finds a difference between N95s (adjusted odds ratio of 0.04) versus other masks (adjusted odds ratio of 0.33) in contracting diseases such as SARS and MERS that workers were unlikely to contract in their daily lives. Overall, 88% protection, i.e. **a factor of 1/8x from a well-sealed N95 mask** seems supported by the data, acknowledging that this could be as little as 1/5x (80% filtration) if there is some leakage, or as much as 1/20x (95% filtration) in cases of perfect usage. We note this is slightly more pessimistic than the [Respirator assigned protection factors](https://en.wikipedia.org/wiki/Respirator_assigned_protection_factors) for this type of PPE, explained in more detail in the next paragraph.
```

Comment:
I think it would be easier to read if we took all the sources we _aren't_ using and put them at the end. It's a lot of mental effort to read "this is the outcome of this study" only to see "but we are ignoring that because xyz." That's 3 pieces of information that are all tangentially related (at best) to the point we are making.

Reply:
fine by me!

Reply:
@beshaya do you wanna open a separate issue for this? Or do you wanna put it in this PR?